src/bot/commands/bonus/member.ts

```diff
@@ -47,7 +47,7 @@ export const member = subcommand({
 
     await statFlushCache.addBonus(member, change);
     await interaction.reply({
-      content: `Successfully gave \`${change}\` bonus XP to ${member}!`,
+      content: `${t(bonus.success)} \`${change}\` ${t(bonusXP)} ${member}!`,
```

Comment:
Wouldn't it be more versatile to be able to move around keywords while translating this? Some languages have different verb order.
```suggestion
      content: t('bonus.success', { change, member }),
```
*this would change `content.json` to*
```json
"success": "Successfully gave/took {{change}} bonus XP to {{member}}"
```